README.md

```diff
@@ -134,6 +135,90 @@ import "@asyncapi/react-component/styles/default.css";
 import "@asyncapi/react-component/styles/default.min.css";
 ```
 
+### Custom styles
+The asyncapi react-component does not set any global fonts. This provides the option to define your custom font-family and related font styling.
+This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using asyncapi react-component.
+
+Custom font styling (like used in AsyncAPI studio): `styles/custom.css` >>
```

Comment:
```suggestion
Example custom styles (used in the [AsyncAPI studio](https://github.com/asyncapi/studio)):

```

Reply:
I refer to the "styles/custom.css" a bit later, so perhaps we can keep it?

Reply:
I thought the user would understand that this name refers to the custom styles, but you could add information that the file is called `styles/custom.css` like:

```md
Example custom styles (defined in the `styles/custom.css` file):
```

Also we don't need to add info where that style is used.

Reply:
Agreed, I had included the "AsyncAPI studio" style since that was the style that is used in the custom example.

I have updated the readme and removed the AsyncAPI studio part.